build.sbt

```diff
@@ -60,8 +60,10 @@ lazy val commonSettings = Seq(
   fork in Test := true
 )
 
+managedSourceDirectories in Compile += target.value / "protobuf-generated"
```

Comment:
Why?

Reply:
https://github.com/sbt/sbt/issues/1583#issuecomment-262166588
This fixed a race condition between ProtoBuf and BuildInfo, I'll add a comment so we remember.